website/static/css/front-page.css

```diff
@@ -0,0 +1,545 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h1,
```

Comment:
I see that the header *tags* need the rules, but are the `.h1`, `.h2`, etc. *classes* actually used? I don't see them anywhere.